src/test/java/teammates/test/cases/logic/InstructorsLogicTest.java

```diff
@@ -728,4 +730,25 @@ private void verifySameInstructor(InstructorAttributes instructor1, InstructorAt
         assertEquals(instructor1.email, instructor2.email);
     }
 
+    private void testGetCoOwnersForCourse() {
+        ______TS("Verify co-owner status of generated co-owners list");
+        String courseId = "idOfTypicalCourse1";
+        List<InstructorAttributes> generatedCoOwners = instructorsLogic.getCoOwnersForCourse(courseId);
+        for (InstructorAttributes generatedCoOwner : generatedCoOwners) {
+            assertTrue(generatedCoOwner.hasCoownerPrivileges());
+        }
+
+        ______TS("Verify all co-owners present in generated co-owners list");
+
+        List<InstructorAttributes> instructors = instructorsLogic.getInstructorsForCourse(courseId);
```

Comment:
@leeyimin New problem with this. The instructors aren't loaded in the same order from both places. As a result the `toString().equals()` check returns a false negative. Going to have to incorporate a sort method to sort the instructors lists by name or key or any other attribute. Do you have any suggestions for a better way to check if an element is present in both ArrayLIsts of InstructorAttributes? (Note, `.contains()` won't work on the type `InstructorAttributes` because it will be checking for memory addresses)